modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-impl/src/main/java/com/liferay/headless/commerce/admin/catalog/internal/dto/v1_0/converter/ProductOptionDTOConverter.java

```diff
@@ -48,7 +49,8 @@ public ProductOption toDTO(DTOConverterContext dtoConverterContext)
 			_cpDefinitionOptionRelService.getCPDefinitionOptionRel(
 				(Long)dtoConverterContext.getId());
 
-		CPOption cpOption = cpDefinitionOptionRel.getCPOption();
+		CPOption cpOption = _cpOptionLocalService.fetchCPOption(
```

Comment:
This should probably go inside the setOptionId() method below because used only there